### DISCUSSION TITLE
Unwrapping

### DISCUSSION THREAD

**Discussion by Abdallahbalushi** [gmtsar/gmtsar#846] [Q&A]
I am trying to do the unwrapping step for a sample data with 6 interferograms. When preparing to do stack.csh and then the next steps before undertaking the unwrapping. It is not working obviously. Should the command be run from the F2 directory or should be run from each interferograms directory. Note: I have not done any merging as I am just test one single subswath.
![image](https://github.com/gmtsar/gmtsar/assets/150544398/c0477f46-5271-4a92-997c-eafd9ba96e7c)
![image](https://github.com/gmtsar/gmtsar/assets/150544398/43f5ac39-c940-4cec-aac2-d9a34897f6fc)

**Comment by Xiaohua-Eric-Xu** [comment]
I think it should be running in your F2/intf_all directory.
